x-pack/plugins/monitoring/public/lib/setup_mode.tsx

```diff
@@ -209,6 +211,7 @@ export const initSetupModeState = async ($scope: any, $injector: any, callback?:
 };
 
 export const isInSetupMode = (context?: ISetupModeContext) => {
+  if (isReactMigrationEnabled()) return setupModeReact.isInSetupMode(context);
```

Comment:
This method is used in many places in react child components rendered also in angular. This is a hack to make it work with the new file without modifying several components.